src/views/Logs/DeconfigurationRecords/DeconfigurationRecords.vue

```diff
@@ -272,8 +272,8 @@ export default {
           sortable: true,
         },
         {
-          key: 'pelID',
-          label: this.$t('pageDeconfigurationRecords.table.pelId'),
+          key: 'EventID',
```

Comment:
Keep this 'E' in lower case, Both here and also in store. As it is variable.

Reply:
Addressed